GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govukfileuploadsettings.config

```diff
@@ -30,68 +30,68 @@
 	<Structure />
 	<GenericProperties>
 		<GenericProperty>
-			<Key>cac445e1-9d87-452e-94de-a70e43ff09ff</Key>
-			<Name>Allowed file types</Name>
-			<Alias>errorMessageAllowedFileExtensions</Alias>
+			<Key>02492b18-6710-4c30-872c-8019f4ba037c</Key>
+			<Name>File types</Name>
+			<Alias>ErrorMessageAllowedFileTypes</Alias>
```

Comment:
I can see you've fixed this but you need to re-export the uSync data for the example app as it still has the old version in page data. Otherwise that change will creep into someone's unrelated PR.

Reply:
Did a Clean Export of the ExampleApp and copied the `FileUploadsettings.config` to the `GovUk.FrontEnd.Umbraco` project

Reply:
It still didn't update the file upload page, but I've done that and will approve this PR.